store/tikv/region_request.go

```diff
@@ -178,6 +186,34 @@ func (s *RegionRequestSender) sendReqToRegion(bo *Backoffer, ctx *RPCContext, re
 	return
 }
 
+func (s *RegionRequestSender) getStoreToken(store *Store) error {
+	for {
+		count := atomic.LoadUint32(&store.tokenCount)
+		if count < atomic.LoadUint32(&config.GetGlobalConfig().TiKVClient.StoreLimit) {
+			if atomic.CompareAndSwapUint32(&store.tokenCount, count, count+1) {
+				return nil
+			}
+		} else {
+			metrics.GetStoreLimitErrorCounter.WithLabelValues(store.addr, string(store.storeID)).Inc()
```

Comment:
1. `store.addr` and `store.storeID` both is unique, we can only need one.
2. You cannot use  `string(store.storeID)` to convert  `int` to `string`

Reply:
1: When DBA finds some storeID was error, the address can help him to locate the machine quickly.  The usage is the same as `GRPCConnTransientFailureCounter`.
2: Addressed.